libsplinter/Cargo.toml

```diff
@@ -99,7 +99,7 @@ experimental = [
     "stable",
     # The following features are experimental:
     "admin-service-event-store",
-    "admin-service-event-store-diesel",
+    "admin-service-event-store-postgres",
```

Comment:
libsplinter/src/store should also be updated for this feature rename

Reply:
Updated the `admin-service-event-store-diesel` instances with a combo of the `admin-service-event-store` guarding the sqlite store functions, and `admin-service-event-store-postgres` in the postgres store implementation